lib/acme_server/standalone.ex

```diff
@@ -11,10 +11,12 @@ defmodule AcmeServer.Standalone do
 
   defp port(Cowboy, adapter_opts), do: cowboy_port(adapter_opts)
   defp port(Cowboy2, adapter_opts), do: cowboy_port(adapter_opts)
+  defp port(Plug.Cowboy, adapter_opts), do: cowboy_port(adapter_opts)
 
   defp cowboy_port(adapter_opts),
     do: adapter_opts |> Keyword.fetch!(:options) |> Keyword.fetch!(:port)
 
   defp adapter_spec(Cowboy, adapter_opts), do: Cowboy.child_spec(adapter_opts)
   defp adapter_spec(Cowboy2, adapter_opts), do: Cowboy2.child_spec(adapter_opts)
+  defp adapter_spec(Plug.Cowboy, adapter_opts), do: Plug.Cowboy.child_spec(adapter_opts)
```

Comment:
The problem here is that you get warnings because we don't depend on `plug_cowboy`. A simple solution could be to add `plug_cowboy` as an optional dependency.